### PR TITLE
page title and svg sprite name updated

### PR DESCRIPTION
**Pull request by srinivashappy**
{% set svgName = "all-sprt" %}
{% set pagetitlehere = "Home" %}

### CODE REVIEW

File: templates-pre/index.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+{% set svgName = "all-sprt" %}
+{% set pagetitlehere = "Home" %}
```
Comment (kamlekar) [inline templates-pre/index.html]: Can you please change this variable name to something relevant?
Like `pageTitle`.

**Comment by srinivashappy** [comment]
{% set svgName = "all-sprt" %}
{% set pagetitle = "Home" %}

**Comment by kamlekar** [comment]
make `t` caps in `pagetitle`. Like this `pageTitle`. and commit the code.

**Comment by srinivashappy** [comment]
{% set svgName = "all-sprt" %}
{% set pageTitle = "Home" %}

**Pull request closed without merging**